### PR TITLE
SwapSort.cpp

### PR DESCRIPTION
**Pull request by mishraabhinn**
#### Checklist

- [ ] Added description of change
- [ ] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTION.md#New-File-Name-guidelines)
- [ ] Added tests and example, test must pass
- [ ] Relevant documentation/comments is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTION.md#Commit-Guidelines)
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] Sort by alphabetical order
- [ ] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

**Comment by mishraabhinn** [comment]
Done @ayaankhan98 . Now I think it looks good.

**Comment by ayaankhan98** [comment]
@cclauss your feedback ?

**Comment by cclauss** [comment]
Filename must end in .cpp

**Comment by ayaankhan98** [comment]
yes rename it to `swap_sort.cpp`, earlier i had not looked over the filename.

**Comment by mishraabhinn** [comment]
Done @cclauss.

**Comment by cclauss** [comment]
Please look at the failing `cpplint_modified_files` tests next the the ❌ below.

TL;DR lines that end with witespace and lines indented with tabs instead of four spaces fail our lint tests.   You can fix these manually or run [`clang-format`](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTION.md#pull-requests) to autofix.

**Comment by mishraabhinn** [comment]
Ty @ayaankhan98 for suggesting.

**Comment by ayaankhan98** [comment]
@mishraabhinn consider this error 
`sorting/SwapSort.cpp:1:  Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]`

**Comment by mishraabhinn** [comment]
clang-format -i -style="{IndentWidth: 4}" my_file.cpp
I have used this command.How is the error coming @ayaankhan98

**Comment by cclauss** [comment]
Whitespace at the end of line 1
`sorting/SwapSort.cpp:1:  Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]`
